HelpSource/Reference/Syntax-Shortcuts.schelp

```diff
@@ -150,18 +220,32 @@ table::
 ## code:: Array.series(16,1,1) :: or code:: series(1,nil,16) :: || code:: (1..16) ::
 ## code:: Array.series(6,1,2) :: or code:: series(1,3,11) :: || code:: (1,3..11) ::
 ::
-There is also the similar syntax for creating an iterating link::Classes/Routine:: :
+There is also the similar syntax for creating an iterating link::Classes/Routine:::
 table::
 ## instead of writing: || you can write:
 ## code:: seriesIter(1,3,11) :: || code:: (:1,3..11) ::
 ::
+note::
+SuperCollider also supports link::Guides/ListComprehensions:: (actually their generator Routines).
```

Comment:
did you intend "they're" here? i'm not sure i understand the parenthetical

Reply:
Well, *list* comprehensions in SC don't exist per se. The `{: }` only returns a Routine. In order to get the actual list you have to invoke `all` on the routine returned by `{: }`. In Python such routines are [called](https://code-maven.com/list-comprehension-vs-generator-expression#:~:text=A%20Generator%20Expression%20is%20doing,immediately%20and%20returns%20a%20list.) "generators"

Reply:
since the terms are not really well defined in SC and the purpose of this PR is just to improve documentation, i think dropping the parenthetical would be more expedient

Reply:
did it.